tools/IceRpc.Slice.Tools/IceRpc.Slice.Tools.targets

```diff
@@ -73,16 +73,11 @@
     </SliceCCSharpTask>
 
     <!-- Run build telemetry -->
-    <BuildTelemetryTask
+    <Exec
       WorkingDirectory="$(IceRpcBuildTelemetryScriptPath)"
-      CompilationHash="$(Hash)"
-      ContainsSlice1="$(ContainsSlice1)"
-      ContainsSlice2="$(ContainsSlice2)"
-      SourceFileCount="$(SourceFileCount)"
-      ReferenceFileCount="$(ReferenceFileCount)"
+      Command="dotnet IceRpc.BuildTelemetry.Reporter.dll --idl slice --hash=$(Hash) --contains-slice1=$(ContainsSlice1) --contains-slice2=$(ContainsSlice2) --src-file-count=$(SourceFileCount) --ref-file-count=$(ReferenceFileCount)"
```

Comment:
Is there a reason why we use `--switch=value` for everything except for the `idl`?
(there's no equal sign for that one).

I guess it doesn't make a difference whether you put an equals or not, but the old `BuildTelemetryTask` didn't place any `=` between switches and their values.

Reply:
Removed `=`